openedx/core/lib/xblock_builtin/xblock_discussion/static/js/discussion_classes.js

```diff
@@ -0,0 +1,23 @@
+// Add RequireJS definitions for each discussion class
+
```

Comment:
@e-kolpakov Good point about RequireJS support in Apros, I'll have to check what the situation is.